css/CalendarMonth.scss

```diff
@@ -14,6 +14,9 @@
   table {
     border-collapse: collapse;
     border-spacing: 0;
+    caption: {
+      caption-side: top;
```

Comment:
Since we don't currently specify a caption-top, I'd think we want the browser default.